packages/@okta/pseudo-loc/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "pseudo-loc-for-signin-widget",
+  "version": "1.0.0",
+  "description": "Generates pseudo-loc for the Sign-In Widget",
+  "scripts": {
+    "pseudo-loc": "node pseudo-loc.js"
+  },
+  "devDependencies": {
+    "@okta/i18n.pseudo-loc": "0.0.0-4584-gced6ef0"
```

Comment:
Can you add a small comment around what the steps look like when we want to upgrade `@okta/i18n.pseudo-loc`. Is it just updating this value and running install again ?

Reply:
Yep - that is the gist of it. Preferably, updating this dependency will be handled by UI Core alone, as we'll want parity between all `i18n.pseudo-loc` packages.

Given that, I don't think we need to worry about upgrade instructions. How about you?

Reply:
Ok sounds good. I think for the near future UI will be involved in the upgrade.